brevbaker-dsl/src/main/kotlin/no/nav/pensjon/brev/template/IncludeAttachment.kt

```diff
@@ -20,14 +22,29 @@ fun TextScope<BaseLanguages, *>.namedReference(attachment: AttachmentTemplate<Ba
     text(Language.Bokmal to "»", Language.Nynorsk to "»", Language.English to "”")
 }
 
-data class IncludeAttachment<out Lang : LanguageSupport, AttachmentData : Any>(
+class IncludeAttachment<out Lang : LanguageSupport, AttachmentData : Any> @InternKonstruktoer constructor(
```

Comment:
Hva er grunnen til å ikke bruke `internal`?

Reply:
Merka at vi generelt bruker mange av dsl-klassane også i biblioteks-modulen, og nokre av dei også i selectors, så da gjekk eg for konsekvent annotasjons-løysing. Men kanskje er det betre med internal der vi kan ha det, og annotasjon der vi må 🤔

Reply:
Spørsmålet gjelder egentlig alle eller i hvertfall mange steder denne annotasjonen brukes.

Reply:
Nokre stadar (feks Argument, UnaryInvoke og SafeCall) må vi ha det fordi vi generer selectors som bruker det.

Andre stadar må vi ha det fordi vi har etterslep med å få flytta/splitta testar (TemplateListTest, TemplateTableTest og TemplateTest bør vi openbart få flytta over frå biblioteket og hit, da kan vi gjera mykje av Element-strukturen til vanleg `internal constructor`.

Andre stadar (feks Element.NewLine) har eg brukt annotasjonen heller enn `internal constructor` for å vera konsekvent.

Eigentleg er det jo fint viss vi klarer å innkapsle mest mogleg av dette i dsl-modulen.